rpc/common/TransactionRequestBatcher.java

```diff
@@ -0,0 +1,170 @@
+/*
```

Comment:
I wonder if this class is still simple enough for us to reasonably expect an external contributor to replicate it in another language.

Perhaps there's value in compiling some BDD concurrency scenarios that could easily fail with a botched client implementation. Although I don't imagine this will be a priority anytime soon.